adhocracy4/comments/serializers.py

```diff
@@ -55,14 +55,14 @@ def to_internal_value(self, data):
 
     def get_user_name(self, obj):
         """
-        Don't show username if comment is marked removed or censored
+        Don't show username if comment is marked removed or censored or blocked
         """
-        if(obj.is_censored or obj.is_removed):
+        if obj.is_censored or obj.is_removed or obj.is_blocked:
```

Comment:
Hmm, unsure here. If we don't want to care about the new field in the old comments, then we also shouldn't take it into account here? Or do we just leave it in case someone sets it in the django admin?

Reply:
Yeah, no, I would remove it. Otherwise it's a bit weird, that the field isn't in the serializer, but still influences the other fields. Do you agree @Rineee and @sabinammm ?

Reply:
Yeah, I was also unsure here.. but you are probably right, even if someone sets it in django-admin, it shouldnt mean anything, so lets delete that here.

Reply:
Yes, that makes sense because it's in the old comments. I wasn't sure how to deal with them anyway, but would take out everything that has to do with them anyway.